manifests/config/worker.pp

```diff
@@ -63,8 +64,25 @@
   if $pool_create {
     $condor_directories = [
       $pool_home,
-      "${pool_home}/condor",
       '/etc/condor/persistent']
+    file { "${pool_home}/condor":
+      ensure  => directory,
+      owner   => 'condor',
+      mode    => '0644',
+      seltype => 'condor_var_lib_t',
```

Comment:
I am not familiar with the selinux module, but since `seltype` is repeated later one, does it need to be set at this stage?

Reply:
Yes, both are necessary. 
One is used to assign the correct context when the file resource is ensured (if nothing is passed, puppet will reset the `seltype` to the unconfined default). 
The second instance (`selinux::fcontext`) informs SELinux that this `seltype` should be used for all directories matching the given pattern, so a future run of `restorecon` will not roll back any defined contexts. 
If only one of the two would be set (I had this initially), we end up in a flip-flop situation whenever `restorecon` is called anywhere else.

Reply:
Thanks for the explanation!